Tests/SwiftDocCTests/DocumentationService/ConvertService/ConvertServiceTests.swift

```diff
@@ -355,10 +355,25 @@ class ConvertServiceTests: XCTestCase {
                 "/documentation/MyKit/MyClass-swift.class/myFunction()-swift.method"
             )
             
-            XCTAssertEqual(
-                renderNode.abstract?.first,
-                .codeVoice(code: "myFunction()")
-            )
+            if LinkResolutionMigrationConfiguration.shouldUseHierarchyBasedLinkResolver {
```

Comment:
for this test and the others, would be set at the command-line when invoking testing? Should we have a variant of this test that runs with `shouldUseHierarchyBasedLinkResolver == true` so that we test both implementations? Otherwise we might miss regressions with one or other implementation. We can extract the test method body into a separate function and call it in different tests with `shouldUseHierarchyBasedLinkResolver` true or false.

Reply:
It's possible to run the tests using the new implementation by setting `DOCC_USE_HIERARCHY_BASED_LINK_RESOLVER=YES` as an environmental variable. 

I think that It's equally important to catch regressions in the tests that have the same behavior in both implementations as it is in the handful of tests that have differences in behavior across the two implementations. 

If the time increase is acceptable I would suggest updating "bin/test" to run all the tests with both implementations. It not, then maybe we can run the new implementation before merging for PRs where we think it's needed and at some point flip the default value so that the new implementation always runs and we'd run the previous implementation before merging PRs.